### PR TITLE
Add tree-sitter-rec for RecUtils

### PR DESCRIPTION
**Pull request by thmsmlr**
I recently put together a tree sitter implementation for [RecUtils](https://www.gnu.org/software/recutils/), I figure we could add it here for others to use

### CODE REVIEW

**Review by kyazdani42 — COMMENTED** [review]
Would you mind adding a few queries for it ? Or will you do that later ?

File: lua/nvim-treesitter/parsers.lua [hunk 1]
```diff
@@ -1068,6 +1068,15 @@ list.proto = {
   filetype = "proto",
 }
 
+list.rec = {
+  install_info = {
+    url = "https://github.com/thmsmlr/tree-sitter-rec",
+    files = { "src/parser.c" },
```
Comment (theHamsta) [inline lua/nvim-treesitter/parsers.lua:1074]: you have to add `branch = "main"` to make it compile

**Comment by theHamsta** [comment]
The grammar is very simple (https://github.com/thmsmlr/tree-sitter-rec/blob/main/grammar.js). Is this already supported by Vim's regex highlighting. With those few nodes, I suppose there would only be additional value when we have queries for that language

**Comment by theHamsta** [comment]
Are there any plans to add queries or can we close?

**Comment by clason** [comment]
> Are there any plans to add queries or can we close?

guess not... 😞

**Pull request closed without merging**